services/viva-ms/lambdas/syncWorkflow.js

```diff
@@ -95,8 +120,7 @@ async function addWorkflowToCase(PK, SK, workflow) {
 
   const [updateError] = await to(dynamoDb.call('update', params));
   if (updateError) {
-    console.log('(Viva-ms) syncWorkflow update error', updateError);
-    return false;
+    return console.error('(Viva-ms) syncWorkflow', updateError);
```

Comment:
console.error returns void, so this return seems strange, especially since you have 'return true' further down. Either log the error and return false, or don't return anything (i.e. also remove the return true further down).

Reply:
Just trying to be consistent here with the rest of the code base. I have seen the same structure where we return console.error but I would prefer to return true / false and log errors.
Should I be consistent or break the pattern and return bool?

Reply:
In my opinion returning a console.log or console.error feels strange and unclear, since you are really returning void in a weird way. Especially strange if you elsewhere return boolean values, then you return different types of values depending on the evaluation. But perhaps someone else (i.e. Teddy and/or Dan) has some ideas about this.

Reply:
After som reading in the AWS docs my conclusion is that it's completely fine to use throw syntax. So what you could do @zilaei is either throw an error or a warning ie:
throw new Error('error message') or throw 'Some information'

Have not tested what comes out in the logs, but adding a console log before throwing is fine in my opinon (y)